dbcon/mysql/ha_mcs.cpp

```diff
@@ -1155,11 +1682,26 @@ maria_declare_plugin(columnstore)
   columnstore_init_func,
   columnstore_done_func,
   MCSVERSIONHEX,
-  mcs_status_variables,          /* status variables                */
-  mcs_system_variables,          /* system variables                */
+  mcs_status_variables,          /* status variables */
+  mcs_system_variables,          /* system variables */
   MCSVERSION,                    /* string version */
   COLUMNSTORE_MATURITY           /* maturity */
 },
+{
+  MYSQL_STORAGE_ENGINE_PLUGIN,
+  &ha_mcs_cache_storage_engine,
+  "Columnstore_cache",
+  "MariaDB Corporation AB",
+  "Insert cache for ColumnStore",
+  PLUGIN_LICENSE_GPL,
+  ha_mcs_cache_init,            /* Plugin Init */
+  ha_mcs_cache_deinit,          /* Plugin Deinit */
+  0x0100,                       /* 1.0 */
```

Comment:
We should align the version with MCS version.

Reply:
Ok. I have changed this to use MCS versions here.